.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - travis_retry composer install --no-progress --prefer-dist
 script:
   # Tests
+  - mkdir tests/acceptance tests/functional || true
```

Comment:
imho It's better to create directories and add .gitkeep

Reply:
@MartkCz Fixed